com.unity.shadergraph/Editor/Drawing/PreviewManager.cs

```diff
@@ -354,11 +354,11 @@ void AssignPerMaterialPreviewProperties(Material mat, List<PreviewProperty> perM
                             {
                                 // Ensure we always request the mip sized 256x256
                                 int width, height;
-                                UnityEngine.Rendering.VirtualTexturing.System.GetTextureStackSize(mat, stackPropertyId, out width, out height);
+                                UnityEngine.Rendering.VirtualTexturing.Streaming.GetTextureStackSize(mat, stackPropertyId, out width, out height);
```

Comment:
This change is part of another PR and should not be included here. Please remove so this PR is confined to HD only.

Reply:
This diffe should now be gone after merging master.